java-extensions/udf-examples/src/main/java/com/starrocks/example/udf/README.md

```diff
@@ -18,3 +18,34 @@ usage:
 > select udf_add(NULL, 1);
 NULL
 ```
+
+
+
+# UDF JSON_AGG
+
+add two parameter
+```
+CREATE AGGREGATE FUNCTION json_agg(String, String)
+RETURNS string
+properties (
+	"symbol" = "com.starrocks.example.udf.JsonAgg",
+	"type" = "StarrocksJar",
+	"file" = "http://$HTTP_HOST:$PORT/xxx.jar"
+);
+```
+#### usage:
+```
+mysql>select db_udf.json_agg(columnA, columnB)
```

Comment:
please add more explain for this udf,  such as why c1 value is 333 not 111, and this udf could use in which case. Thanks.